creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.ts

```diff
@@ -422,7 +429,7 @@ const _additionalSyncIsRequired = async (
   const startTimeMs = Date.now()
   const maxMonitoringTimeMs =
     startTimeMs +
-    (syncType === SyncType.MANUAL
+    (syncType === SyncType.Manual
```

Comment:
This was a bug - there is no such thing as `SyncType.MANUAL`. We've apparently never enforced the timeout here

Reply:
good catch

Reply:
oof - good find

Reply:
lets move this to its own PR and get that out now tho